README.md

```diff
@@ -308,42 +331,53 @@ This returns all users this user has relations with through at least 1 group.
 * 404 if no user with this ID exists or if this user is not the logged-in user.
 
 
-### PUT `/users/{id}`
+### PATCH `/users/{id}`
```

Comment:
PATCH is more appropriate than PUT when we want to change only part of the resource. (and actually I should have used this to push the whole resource anyway, because of `currentPassword`)